modules/tensor_mechanics/src/kernels/lagrangian/LagrangianStressDivergenceBase.C

```diff
@@ -0,0 +1,140 @@
+#include "LagrangianStressDivergenceBase.h"
```

Comment:
I thought `.C` files also need license header. Did it change over the past few months? I'm surprised that the testing system didn't catch this.

Reply:
Yeah, they should be added.

Reply:
Added to the new *.C files.